interfaces/coppeliasim/DQ_CoppeliaSimInterface.m

```diff
@@ -0,0 +1,117 @@
+        stop_simulation(obj);
+
+        % This method gets the handles for a cell array of 
+        % object names in the the CoppeliaSim scene.
+        handles = get_handles(obj,names);
+
+        % This method gets the handle for a given object in the CoppeliaSim scene. 
+        handle = get_handle(obj,name);
+
+        % This method gets the translation of an object in the CoppeliaSim scene.
+        t = get_object_translation(obj,objectname);
+
+        % This method sets the translation of an object in the CoppeliaSim scene.
+        set_object_translation(obj,objectname,translation);
+
+        % This method gets the rotation of an object in the CoppeliaSim scene.
+        r = get_object_rotation(obj, objectname);
+
+        % This method sets the rotation of an object in the CoppeliaSim scene.
+        set_object_rotation(obj,objectname,rotation);
+
+        % This method gets the pose of an object in the CoppeliaSim scene. 
+        x = get_object_pose(obj,objectname);
+
+        % This method sets the pose of an object in the CoppeliaSim scene. 
+        set_object_pose(obj,objectname,pose);
+
+        % This method sets the joint positions of a robot in the CoppeliaSim scene.        
+        set_joint_positions(obj,jointnames,joint_positions);
+
+        % This method sets the joint positions of a robot in the CoppeliaSim scene. 
+        joint_positions = get_joint_positions(obj,jointnames);
+
+        % This method sets the joint target positions of a robot in the CoppeliaSim scene. 
+        % It is required a dynamics enabled scene, and joints in dynamic mode 
+        % with position control mode. 
+        % information about joint modes:
+        % https://www.coppeliarobotics.com/helpFiles/en/jointModes.htm
+        set_joint_target_positions(obj,jointnames,joint_target_positions);
+
+        % This method gets the joint velocities of a robot in the CoppeliaSim scene. 
+        joint_velocities = get_joint_velocities(obj,jointnames);
+
+        % This method sets the joint target velocities of a robot in the CoppeliaSim scene.
+        % It is required a dynamics enabled scene, and joints in dynamic mode 
+        % with velocity control mode. Check this link for more
+        % information about joint modes:
+        % https://www.coppeliarobotics.com/helpFiles/en/jointModes.htm
+        set_joint_target_velocities(obj,jointnames,joint_target_velocities);
+
+        % This method sets the joint torques of a robot in the CoppeliaSim scene.
+        % It is required a dynamics enabled scene, and joints in dynamic mode 
+        % with velocity or force/torque control mode. Check this link for more
+        % information about joint modes:
+        % https://www.coppeliarobotics.com/helpFiles/en/jointModes.htm
+        set_joint_torques(obj,jointnames,torques);  
+
+        % This method gets the joint torques of a robot in the CoppeliaSim scene.
+        joint_torques = get_joint_torques(obj,jointnames);
+    end
```

Comment:
@ffasilva, thank you for the reminder. 

Note for the future Juancho (since I am going to completely forget this): we discussed this internally. We decided to add those methods in a future PR since they are not available in the master version of the DQ Robotics. We are still working on it at https://github.com/dqrobotics/matlab/pull/109